dojo/tools/factory.py

```diff
@@ -304,6 +305,8 @@ def import_parser_factory(file, test, active, verified, scan_type=None):
         parser = OrtParser(file, test)
     elif scan_type == 'SARIF':
         parser = SarifParser(file, test)
+    elif scan_type == 'OssIndex Devaudit SCA Scan Results':
```

Comment:
There is some inconsitancies in the PR for the values of parser identifier:
* `OssIndex Devaudit SCA Scan`
* `OssIndex DevAudit scan result importer`
* `OssIndex Devaudit SCA Scan Results`

Those 3 values should be the same.

Reply:
Fixed all three locations using "OssIndex Devaudit SCA Scan Importer"